tools/buckconfigs/freebsd-x86_64/type/debug.bcfg

```diff
@@ -1,5 +1,5 @@
 [project]
-  buck_out = "buck-out/debug"
+  buck_out = buck-out/debug
```

Comment:
Are you sure about this? I seem to remember that (at least in some environments) the quotes were required (and on others optional). Can you add some info about why this is needed for future reference?

Reply:
I’ll double check. The quotes are breaking macOS right now, on my machine and on Travis so this change is at least needed there.